doc/api/cli.md

```diff
@@ -2018,6 +2018,31 @@ On a machine with 2 GiB of memory, consider setting this to
 $ node --max-old-space-size=1536 index.js
 ```
 
+### `--max_semi_space_size=SIZE` (in megabytes)
+
+Sets the maximum
+[semi-space](https://www.memorymanagement.org/glossary/s.html#semi.space)
+size for V8's [scavenge](https://v8.dev/blog/orinoco-parallel-scavenger)
+garbage collector in MiB.
+Increasing the semi-space size by 1MiB will
+cause the heap size of V8 to increase by 3MiB,
+but the throughput improvement depends on your workload
+(see [#42511](https://github.com/nodejs/node/issues/42511)).
+
+The default value is
+16MiB for 64-bit systems and 8MiB for 32-bit systems.
+To get the best configuration for your application,
+you should try different max\_semi\_space\_size values
```

Comment:
Use dashes (`-`) for consistency with `cli.md`.
> All options, including V8 options, allow words to be separated by both dashes (`-`) or underscores (`_`). For example, `--pending-deprecation` is equivalent to `--pending_deprecation`.
_(src: https://nodejs.org/api/cli.html#options)_

---
```suggestion
you should try different max-semi-space-size values
```